### PR TITLE
doc: Adding TUNNEL_DECAP schema

### PR DESCRIPTION
**Pull request by scsmncao**
Adding the TUNNEL_DECAP_TABLE schema to swss-schema.md

### CODE REVIEW

File: doc/swss-schema.md [hunk 1]
```diff
@@ -270,6 +270,17 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
 	4) "30720"
 
 ----------------------------------------------
+###TUNNEL_DECAP_TABLE
+    ; Stores tunnel decap rules
+    ; Status: Work in progress
```
Comment (lguohan) [inline doc/swss-schema.md]: remove this "status" line.

File: doc/swss-schema.md [hunk 2]
```diff
@@ -270,6 +270,17 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
...
+    key                     = TUNNEL_DECAP_TABLE:name
+    tunnel_type             = "IPINIP"
+    dst_ip                  = IP1,IP2 ;IP addresses separated ","
+    dscp_mode               = "uniform" / "pipe"
+    ecn_mode                = "copy_from_outer" / "standard"
+    ttl_mode                = "uniform" / "pipe" / "origin" ;origin means keeping the TTL in the inner packet header. Not yet defined in SAI.
```
Comment (lguohan) [inline doc/swss-schema.md]: remove origin, the pipe mode

Comment (lguohan) [inline doc/swss-schema.md]: remove "origin" model

File: doc/swss-schema.md [hunk 3]
```diff
@@ -270,6 +270,17 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
 	4) "30720"
 
 ----------------------------------------------
+###TUNNEL_DECAP_TABLE
+    ; Stores tunnel decap rules
+    ; Status: Work in progress
+    key                     = TUNNEL_DECAP_TABLE:name
+    tunnel_type             = "IPINIP"
+    dst_ip                  = IP1,IP2 ;IP addresses separated ","
```
Comment (lguohan) [inline doc/swss-schema.md]: separated by ","

File: doc/swss-schema.md [hunk 4]
```diff
@@ -270,6 +270,17 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
...
+    dscp_mode               = "uniform" / "pipe"
+    ecn_mode                = "copy_from_outer" / "standard"
```
Comment (lguohan) [inline doc/swss-schema.md]: add comments. ; standard: Behavior defined in RFC 6040 section 4.2

**Comment by lguohan** [comment]
:shipit:

**Comment by stcheng** [comment]
Squashed the changes and rebased the branch.

**Pull request merged**